demos/default/app/app-edge/layout.tsx

```diff
@@ -0,0 +1,11 @@
+'use client'
+
+// TODO-APP: support typing for useSelectedLayoutSegment
+// @ts-ignore
+import { useSelectedLayoutSegments } from 'next/navigation'
```

Comment:
FYI for anyone, https://beta.nextjs.org/docs/api-reference/use-selected-layout-segments